dune_api_scripts/update_appdata_view.py

```diff
@@ -0,0 +1,18 @@
+"""Modifies and executed dune query for today's data"""
+from .utils import dune_from_environment, build_string_for_affiliate_referrals_pairs
+
+if __name__ == "__main__":
+    # initialize the environment
+    dune = dune_from_environment()
+
+    # build query
+    QUERY = f"""CREATE OR REPLACE VIEW
+    dune_user_generated.gp_appdata (app_data, referrer)
+    AS VALUES {build_string_for_affiliate_referrals_pairs()}"""
+
+    # update query in dune
+    QUERY_ID = 257782
```

Comment:
I think its best to get it via an env variable, like here:
`    query_id = int(os.getenv('QUERY_ID_ALL_APP_DATA', "257770"))`

Reply:
We also may need to settle on a single user account.